monitoring/rid_qualifier/utils.py

```diff
@@ -0,0 +1,129 @@
+from typing import List, NamedTuple, Any
+from shapely.geometry import Point, Polygon
+import shapely.geometry
+from datetime import datetime, timedelta
+
+
+class RIDSerializer():
+    ''' A class to serialize RID objects in the qualifier '''
+        
+    def make_json_compatible(self, struct: Any) -> Any:
```

Comment:
I don't think we need to have this method in a class -- it is a pure function and doesn't need an instance context to work properly.

Reply:
OK thanks, no problems, I will move this out. The only thing is that this shared between the different components so I thought I will move it to utils.